### PR TITLE
Prevent errors in qt/test_plugin_widgets from installed plugins

### PR DESCRIPTION
**Pull request by tlambert03**
# Description
small path that makes the test_plugin_widgets test less less prone to fail due to plugins installed in the env.  

can wait till after the release

### CODE REVIEW

File: napari/_qt/_tests/test_plugin_widgets.py [hunk 1]
```diff
@@ -97,18 +97,14 @@ def test_plugin_widgets_menus(test_plugin_widgets, make_napari_viewer):
...
             break
     actions = actions[cnt + 1 :]
-    assert len(actions) == 3
-    expected_text = ['TestP1', 'TestP2: Widg3', 'TestP3: magic']
-    assert [a.text() for a in actions] == expected_text
+    texts = [a.text() for a in actions]
+    for t in ['TestP1', 'TestP2: Widg3', 'TestP3: magic']:
+        assert t in texts
 
     # the first item of the plugins is a submenu (for "Test plugin1")
```
Comment (nclack) [inline napari/_qt/_tests/test_plugin_widgets.py]: Looks like the comment won't be quite right after the change.
```suggestion
    # Expect a submenu ("Test plugin1") with particular entries.
```

**Review by nclack — APPROVED** [review]
Thanks!  

A lot of these changes also happen to make the tests insensitive to menu item sort-order. Is the order that plugins are presented in the menu defined? For example, should there be a test checking that 'TestP1' always comes before 'TestP3'? Maybe one's there and I missed it.

**Comment by tlambert03** [comment]
> Is the order that plugins are presented in the menu defined

No not yet, it's just discovery order.  We can test that when actually try to control it

File: napari/_qt/_tests/test_plugin_widgets.py (hunk 1 above)
Comment (sofroniewn) [reply napari/_qt/_tests/test_plugin_widgets.py]: Should this change be accepted?

**Review by sofroniewn — APPROVED** [review]
LGTM - just left one question above

**Pull request merged**